src/pages/projectSettings/CopyProjectModal.tsx

```diff
@@ -37,14 +37,7 @@ export const CopyProjectModal: React.VFC<Props> = ({
   const [copyProject, { called, data, error, loading }] = useMutation<
     CopyProjectMutation,
     CopyProjectMutationVariables
-  >(COPY_PROJECT, {
-    errorPolicy: "all",
-    refetchQueries: [
-      "GetViewableProjectRefs",
-      "ProjectSettings",
-      "RepoSettings",
-    ],
-  });
+  >(COPY_PROJECT, { errorPolicy: "all" });
```

Comment:
Are we expecting both errors and valid results to be returned?

Reply:
Yes! More context here: https://github.com/evergreen-ci/spruce/pull/1257